airflow/providers/elasticsearch/log/es_task_handler.py

```
@@ -114,6 +115,26 @@ def _ensure_ti(ti: TaskInstanceKey | TaskInstance, session) -> TaskInstance:
         raise AirflowException(f"Could not find TaskInstance for {ti}")
 
 
+def _get_index_patterns(
+    index_patterns_callable: str | None = None, ti: TaskInstance | None = None
+) -> str | None:
+    """
+    Get index patterns by calling index_patterns_callable or None.
+
+    :param index_patterns_callable: A string representing the full path to the callable itself.
+    :param ti: A TaskInstance object or None.
+    """
+    if not index_patterns_callable:
+        return None
+    try:
+        module_path, index_pattern_function = index_patterns_callable.rsplit(".", 1)
+        module = importlib.import_module(module_path)
+        index_pattern_callable_obj = getattr(module, index_pattern_function)
+        return index_pattern_callable_obj(ti)
+    except Exception:
+        return None
```

Happy to return None here, but I don't think we should swallow the exception. If something does fail, it'll be very hard to determine why the expected index_patterns from the callable wasn't used.